src/stats/stats.jl

```diff
@@ -58,6 +61,12 @@ function show(io :: IO, stats :: ExecutionStats)
   println("  feasibility measure: $(stats.feas_norm)")
   println("  iterations: $(stats.iter)")
   println("  elapsed time: $(stats.elapsed_time)")
+  if length(stats.solver_specific) > 0
+    println("  solver specifics:")
+    for (k,v) in stats.solver_specific
+      println("    $k: $v")
```

Comment:
I didn't turn out as I hoped. `show` for vectors defaults to printing all values joined by commas, without space. I was thinking the default could be like NLPModels, so we add an argument for which function to print vectors, and use `show` otherwise.